website/docs/gator.md

```diff
@@ -6,7 +6,7 @@ title: The gator CLI
 The `gator` CLI is a tool for evaluating Gatekeeper ConstraintTemplates and
```

Comment:
i think we need modifications on this file only (which is vNext). For v3.7, we already released it.